File: client/support-tick-client/src/app/models/ticket.ts
export class Ticket {
    constructor(
        public ticketId: number | null,
        public fullName: string,
        public email : string,
        public statuId : number | null,
        public description: string,
        public summary: string | null,
        public imageUrl: string | null,
        public solution: string | null
    ){}
}

File: server/SupportTick/Models/Ticket.cs
namespace SupportTick.Models
{
    public class Ticket
    {
        public int TicketId { get; set; }
        public string FullName { get; set; }
        public string Email { get; set; }
        public int StatuId { get; set; }
        public string Description { get; set; }
        public string? Summary { get; set; }
        public string? ImageUrl { get; set;}
        public string? Solution { get; set;}

    }
}
